src/formats/PDB.cpp

```diff
@@ -299,6 +304,21 @@ void PDBFormat::write(const Frame& frame) {
         auto& type = frame.topology()[i].type();
         auto& pos = frame.positions()[i];
 
+        std::string atom_hetatm;
+        auto atom_property = frame.topology()[i].get("is_hetatm");
+        if (atom_property) {
+          try {
+            if (atom_property->as_bool()) {
+              atom_hetatm = "HETATM";
+            } else{
+              atom_hetatm = "ATOM  ";
+            }
+          }
+          catch (...) {
```

Comment:
why the catch(...) ? Is it to catch the fact that the property might not be a boolean value ? In this case, you can catch `PropertyError`, and I would send a warning about the atom having a non-boolean `is_hetatm` property.

Reply:
Like this?
```
catch (const PropertyError &e) {
            warning("\'is_hetatm\' property set to non-bool variable.
            Defaulting to HETATM");
            atom_hetatm = "HETATM";
          }
```

Reply:
Yes =)